_includes/current_members.html

```diff
@@ -0,0 +1,63 @@
+<script>
+  var req = new XMLHttpRequest();
+  req.overrideMimeType("application/json");
+  req.open('GET', 'https://app.doubleunion.org/public_members', true);
```

Comment:
Right now CORS prevents this request from actually succeeding, before cutting over will need to make a change in arooo to allow this request.